src/ResourceManager/Compute/Commands.Compute/Generated/ContainerService/ContainerServiceDeleteMethod.cs

```diff
@@ -96,67 +96,51 @@ protected PSArgument[] CreateContainerServiceDeleteParameters()
         }
     }
 
-    [Cmdlet(VerbsCommon.Remove, "AzureRmContainerService", DefaultParameterSetName = "InvokeByDynamicParameters", SupportsShouldProcess = true)]
-    public partial class RemoveAzureRmContainerService : InvokeAzureComputeMethodCmdlet
+    [Cmdlet(VerbsCommon.Remove, "AzureRmContainerService", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(void))]
```

Comment:
It's based on a void ```Delete``` method, and more importantly, PS supports that!

So, I guess we don't have to resolve this one.